packages/es-client/esDefaults.js

```diff
@@ -1,7 +1,10 @@
 const defaultESScrollSize = 1000;
 const defaultESScrollDuration = '6m';
 
+const granuleSortParams = { sort: [{ timestamp: 'desc' }, { _uid: 'asc' }] };
```

Comment:
The `uid` value is important for the edge case when timestamps are identical to ensure we can continue `search_after` in the middle of a duplicate timestamp range, and we don't lose the portion of records with identical time stamps that exceed the page limit.

Reply:
Moved the sort params in https://github.com/nasa/cumulus/pull/2993/commits/d4233b24baa062cd9921b551180727db379bb546 to be used as the default in `queries.js` (which effectively only adds the `_uid` tiebreaker since `timestamp: 'desc'` was already the default), so `sort_by` and `order` should now be respected properly again.